physics/altitude_pressure.py

```diff
@@ -0,0 +1,40 @@
+"""
+Title : Calculate altitude using Pressure
+
+Description :
+    The below algorithm approximates the altitude using Barometric formula
+
+
+"""
+
+
+def get_altitude_at_pressure(pressure: float) -> float:
+    """
+    This method calculates the altitude from Pressure wrt to
+    Sea level pressure as reference .Pressure is in Pascals
+    https://en.wikipedia.org/wiki/Pressure_altitude
+    https://community.bosch-sensortec.com/t5/Question-and-answers/How-to-calculate-the-altitude-from-the-pressure-sensor-data/qaq-p/5702
+
+    H = 44330 * [1 - (P/p0)^(1/5.255) ]
+
+    Where :
+    H = altitude (m)
+    P = measured pressure
+    p0 = reference pressure at sea level 101325 Pa
+
+    Examples:
+
+    >>> get_altitude_at_pressure(pressure=100000)
+    105.47836610778828
```

Comment:
Please make tests for H ~= sea level, H ~= 100 meters, H ~= 1,750 (altitude of my house) also a zero pressure, a negative pressure, and a pressure greater than 101,325... The last few should prove that the code raises ValueErrors like...
https://github.com/TheAlgorithms/Python/blob/d31750adece86ebf39a09dd3adb2039098f58586/arithmetic_analysis/bisection.py#L9-L12